### DISCUSSION TITLE
Multiple simultaneous broadcast on a Single Station

### DISCUSSION THREAD

**Discussion by nmiller0113** [AzuraCast/AzuraCast#6686] [Q&A] [answered]
I was wondering if there is a way to have a single station with multiple broadcast going simultaneously. Let's say I want one playing jazz and another playing top 40. That way when someone goes to the public page they can see two options? Another reason I'm asking is because I couldn't figure out how to do it and ended up just creating another station, but then realized I couldn't use the same broadcasting port of 8000, which I was able to do when just using Icecast alone. Two mount points, both :8000/jazz and another :8000/top40.

**Comment by BusterNeece** [comment, marked as answer]
This is by design in the AzuraCast architecture. One station corresponds to one set of media, and one composited broadcast that's output to local broadcasting, remote relays, etc...

That's part of why we have the web proxy set up the way we do, so you can still direct users to `/listen/jazz` and `/listen/top40` despite the fact that they're served on different ports (which is necessary to "sandbox" them from one another, as would be necessary in multi-tenant environments).

**Comment by nmiller0113** [comment]
Thanks @BusterNeece, that makes sense! Appreciate the response and really love Azuracast!

**Comment by nmiller0113** [comment]
@BusterNeece sorry, I actually just realized that I only get the icecast page if I do `/listen/jazz`. If I go `/listen/jazz/jazz` then I get to the stream. Perhaps I've got something misconfigured?

**Comment by nmiller0113** [comment]
@BusterNeece I misspoke. It is actually `/listen/jazz/radio.mp3`
